tor/core/config.py

```diff
@@ -0,0 +1,128 @@
+# Load configuration regardless of if bugsnag is setup correctly
+try:
+    import bugsnag
+except ImportError:
+    bugsnag = None
+
+from tor import __version__
+
+
+class BaseConfig:
+    '''
+    '''
```

Comment:
Essentially. It's more along the lines of establishing that all child classes will have a certain interface. It's a kind of throwback from strongly typed, compiled languages which rely on interfaces. Not too useful right now, but it will be with future refactoring.